pontus/week4/app/message-store.js

```diff
@@ -0,0 +1,69 @@
+// var Dispatcher = require('flux').Dispatcher;
+var Constants = require('./constants');
+var BaseStore = require('./base');
+var Dispatcher = require('./dispatcher');
+
+var _messages = [];
+
+let messageStore = Object.assign({}, BaseStore, {
+	getAll() { 
+		return _messages;
+	},
+
+	getOne(messageID) {
```

Comment:
I think it's best to keep one naming convention, either `get()` or `getOne()` in stores.